src/ElmFive.elm

```diff
@@ -181,27 +181,23 @@ translations =
 
 translate : String -> String
 translate language =
-    case Dict.get language translations of
-        Nothing ->
-            "five"
-
-        Just translation ->
-            translation
+    Dict.get language translations
+        |> Maybe.withDefault "five"
 
 
 isFive : Int -> Bool
-isFive value =
-    value == five
+isFive =
+    (==) five
 
 
 filter : List Int -> List Int
-filter values =
-    List.filter (\value -> isFive value) values
+filter =
+    List.filter isFive
```

Comment:
i didn't notice that you excluded `values` in the parameter list. It does indeed work as you indicated :) Thanks!